README.md

```diff
@@ -1,2 +1,27 @@
 # brianhague_dev_be
 Backend for brianhague.dev
+By Brian Hague (alphaseinor)
+
+## Installation Instructions
+[ ] - Fork this repo
+[ ] - Clone your own repo
+[ ] - npm i
+[ ] - touch .env
+[ ] - install postgresql on your local machine and configure .env file for your environment
+[ ] - run knex migrate:latest
+
+## .env file
+DATABASE_URL=postgres://postgres:password@localhost:5432/postgres
```

Comment:
This is a typical local password string, note postgres:password is username:password, and the default db is postgres at the end of the string